### PR TITLE
Adds the ability to delete a transform from etl adapter create view

### PR DESCRIPTION
**Pull request by ajainarayanan**

### CODE REVIEW

File: cdap-ui/app/features/etlapps/controllers/create-ctrl.js [hunk 1]
```diff
@@ -196,6 +196,19 @@ angular.module(PKG.name + '.feature.etlapps')
       });
     };
 
+    $scope.deleteTrasformProperty = function(transform) {
```
Comment (elfenheart) [inline cdap-ui/app/features/etlapps/controllers/create-ctrl.js]: mistype on transform

File: cdap-ui/app/features/etlapps/controllers/create-ctrl.js [hunk 2]
```diff
@@ -196,6 +196,19 @@ angular.module(PKG.name + '.feature.etlapps')
...
+      var filterFilter = $filter('filter');
```
Comment (elfenheart) [inline cdap-ui/app/features/etlapps/controllers/create-ctrl.js]: we don't need this filter anymore right?

File: cdap-ui/app/features/etlapps/etlapps.less [hunk 3]
```diff
@@ -27,13 +27,16 @@ body.theme-cdap.state-adapters-create {
...
     }
-    .fa-pencil {
+    .fa-pencil,.fa-times {
       font-size: 12px;
       vertical-align: middle;
       &:hover {
         cursor: pointer;
       }
     }
+    .fa-times {
```
Comment (elfenheart) [inline cdap-ui/app/features/etlapps/etlapps.less]: Can we move this into the .template.transforms.fa-times? (line 73-76)

File: cdap-ui/app/features/etlapps/controllers/create-ctrl.js [hunk 4]
```diff
@@ -196,6 +196,19 @@ angular.module(PKG.name + '.feature.etlapps')
...
+      var index = $scope.transforms.indexOf(transform);
+      $scope.transforms.splice(index, 1);
```
Comment (elfenheart) [inline cdap-ui/app/features/etlapps/controllers/create-ctrl.js]: We are assuming that we can always find the element. Is there a case where we might not find a match when clicking delete (ie. syncing problem)?

Comment (ajainarayanan) [inline cdap-ui/app/features/etlapps/controllers/create-ctrl.js]: No. We show the list and we delete it. So if the user clicked on the item to be deleted from an array then we are sure that its there.

File: cdap-ui/app/features/etlapps/controllers/create-ctrl.js (hunk 1 above)
Comment (ajainarayanan) [inline cdap-ui/app/features/etlapps/controllers/create-ctrl.js]: Fixed.

File: cdap-ui/app/features/etlapps/controllers/create-ctrl.js (hunk 2 above)
Comment (ajainarayanan) [inline cdap-ui/app/features/etlapps/controllers/create-ctrl.js]: Fixed.

File: cdap-ui/app/features/etlapps/etlapps.less (hunk 3 above)
Comment (ajainarayanan) [inline cdap-ui/app/features/etlapps/etlapps.less]: :+1:

**Comment by elfenheart** [comment]
:+1: LGTM!

**Pull request merged**